scripts/genomics_file_cbio_package_build.py

```diff
@@ -45,6 +45,9 @@ def process_maf(maf_loc_dict, cbio_id_table, data_config_file, dgd_status):
     maf_dir = maf_loc_dict["kf"]
     if args.dgd_status == "dgd":
         maf_dir = maf_loc_dict["dgd"]
+    else:
+        # KF can be in multiple palces
+        maf_dir = ",".join(maf_dir)
```

Comment:
Talked about this at code review but maybe both the DGD and KF maf dir locations should be lists. Would remove the need for handling different data types.

Reply:
I'll keep this in mind when DGD panel data comes back. I might even expand this to other other file types to sort of future-proof this